doc/user/content/sql/subscribe.md

```diff
@@ -29,21 +29,36 @@ You can use `SUBSCRIBE` to:
 
 ## Syntax
 
-{{< diagram "subscribe-stmt.svg" >}}
+```mzsql
```

Comment:
As I finish up with the `SHOW` pages, I thought I'd include the `SUBSCRIBE` as a POC.

Reply:
https://preview.materialize.com/materialize/30689/sql/subscribe/